### PR TITLE
Add Ocular Status

### PR DESCRIPTION
**Pull request by Mbrick2**
Please Squash and Merge

### CODE REVIEW

**Review by ILOVETOMAKESTUFF — CHANGES_REQUESTED** [review]
Please leave the credits as credits, and you should make the ocular status `<i>` tag contain something like "Ocular status not found," because that is what we did for all of the outputs that will be changed

**Review by ILOVETOMAKESTUFF — APPROVED** [review]
nvm

**Comment by webdev03** [comment]
Hello

File: search.js [hunk 1]
```diff
@@ -24,3 +24,14 @@ fetch(`https://periscope-api.vercel.app/api/users/${user}`).then((res) => {
...
   document.getElementById("loading").style.display = "none";
 })
+
+async function getOcularStatus(uname) {
+	url = `https://my-ocular.jeffalo.net/api/user/{uname}`
+	const response = const response = await fetch(url)
+	let data = await response.json();
+	const resObject = JSON.parse(data)
+    const ocular = document.getElementById('ocular-status').innerHTML
+	ocular = resObject['status']
```
Comment (webdev03) [inline search.js:34]: I don't think you can redefine a const??

File: search.js [hunk 2]
```diff
@@ -24,3 +24,14 @@ fetch(`https://periscope-api.vercel.app/api/users/${user}`).then((res) => {
   document.getElementById("info").style.display = "inline";
   document.getElementById("loading").style.display = "none";
 })
+
+async function getOcularStatus(uname) {
+	url = `https://my-ocular.jeffalo.net/api/user/{uname}`
+	const response = const response = await fetch(url)
+	let data = await response.json();
+	const resObject = JSON.parse(data)
```
Comment (webdev03) [inline search.js:32]: It's already parsed with .json this will just throw an error

File: search.js [hunk 3]
```diff
@@ -24,3 +24,14 @@ fetch(`https://periscope-api.vercel.app/api/users/${user}`).then((res) => {
   document.getElementById("info").style.display = "inline";
   document.getElementById("loading").style.display = "none";
 })
+
+async function getOcularStatus(uname) {
+	url = `https://my-ocular.jeffalo.net/api/user/{uname}`
+	const response = const response = await fetch(url)
```
Comment (webdev03) [inline search.js:30]: You repeated yourself

File: search.js [hunk 4]
```diff
@@ -24,3 +24,14 @@ fetch(`https://periscope-api.vercel.app/api/users/${user}`).then((res) => {
   document.getElementById("info").style.display = "inline";
   document.getElementById("loading").style.display = "none";
 })
+
+async function getOcularStatus(uname) {
+	url = `https://my-ocular.jeffalo.net/api/user/{uname}`
```
Comment (webdev03) [inline search.js:29]: This needs to be `${}` not `{}`

File: user.html [hunk 5]
```diff
@@ -6,7 +6,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;600&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
-    <title>Periscope Search | Credits</title>
+    <title>Periscope Search | User</title>
```
Comment (webdev03) [inline user.html:9]: Thanks!! 😄

File: search.js (hunk 1 above)
Comment (webdev03) [reply search.js:34]: I thought you can only change the values in the object. Not sure. Maybe you are right.

**Pull request merged**